### PR TITLE
Translate intervalboundary enum

### PR DESCRIPTION
**Pull request by Fan2Shrek**

### CODE REVIEW

File: reference/random/random.intervalboundary.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  <section xml:id="enum.random.intervalboundary.synopsis">
+   &reftitle.enumsynopsis;
+
+   <enumsynopsis>
+    <enumname>Random\IntervalBoundary</enumname>
+
+    <enumitem>
+     <enumidentifier>ClosedOpen</enumidentifier>
+     <enumitemdescription>
+      Un intervalle gauche-ouvert.
```
Comment (devnexen) [inline reference/random/random.intervalboundary.xml:27]: nit: I feel `une intervalle fermée à gauche` pertains more the of the lower bound inclusion/upper bound exclusion wdyt ?

Comment (Fan2Shrek) [reply reference/random/random.intervalboundary.xml]: Sounds better :)

**Review by devnexen — APPROVED** [review]

File: reference/random/random.intervalboundary.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  <section xml:id="enum.random.intervalboundary.synopsis">
+   &reftitle.enumsynopsis;
+
+   <enumsynopsis>
+    <enumname>Random\IntervalBoundary</enumname>
+
+    <enumitem>
+     <enumidentifier>ClosedOpen</enumidentifier>
+     <enumitemdescription>
+      Un intervalle fermé à gauche.
```
Comment (Girgias) [inline reference/random/random.intervalboundary.xml:27]: "Une intervalle", non?

Comment (devnexen) [reply reference/random/random.intervalboundary.xml:27]: I think it s correct [in this case](https://www.collinsdictionary.com/dictionary/french-english/intervalle) [wdyt](https://www.lalanguefrancaise.com/orthographe/un-intervalle-ou-un-interval) ?.

**Pull request merged**